### DISCUSSION TITLE
why only I is prompted to input password?

### DISCUSSION THREAD

**Discussion by cadem** [community/community#49142] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

There is a private repository of my company in github, ONLY I am required to input password when doing git clone https, no other is required. The company github administrator comfirm my priviledge is correct, and is  same with other people. Is any special setting in my account?

I know how to work around (such as SSH public key or https personal token), but I want to know the reason. Anyone know it?

**Comment by airtower-luna** [comment]
Most likely the other people in your company have their tokens stored *somewhere* in their systems. Git can store credentials in various ways (and you can add your own), see [Git Tools - Credential Storage](https://git-scm.com/book/en/v2/Git-Tools-Credential-Storage). You can compare the settings mentioned there with your colleagues', or just set up a storage option you like.
